snapshot/src/main/java/io/zeebe/snapshots/impl/SnapshotChecksum.java

```diff
@@ -74,20 +75,17 @@ public static boolean verify(final Path snapshotDirectory) throws IOException {
   private static long createCombinedChecksum(final List<Path> paths) throws IOException {
     final Checksum checksum = SnapshotChunkUtil.newChecksum();
 
+    final ByteBuffer buff = ByteBuffer.allocate(IoUtil.BLOCK_SIZE);
```

Comment:
Should we enforce byte order? Possibly if two nodes have different native byte order, then when comparing checksums they would have different checksums?

Reply:
Does it matter? We are not reading non-byte primitive types.

Reply:
I don't know how the checksum interprets them - but I guess you're probably right, it most likely doesn't interpret them but just reads byte by byte.